tests/flytekit/unit/extras/tensorflow/test_native.py

```diff
@@ -0,0 +1,29 @@
+from typing import List
+
+import tensorflow as tf
+
+from flytekit import task, workflow
+
+
+@task
+def generate_model() -> tf.keras.Model:
+    inputs = tf.keras.Input(shape=(32,))
+    outputs = tf.keras.layers.Dense(1)(inputs)
+    model = tf.keras.Model(inputs, outputs)
+    return model
+
+
+@task
+def get_model_layers(model: tf.keras.Model) -> List[tf.keras.layers.Layer]:
+    return model.layers
+
+
```

Comment:
@techytushar, can you also add a test for `tf.keras.Sequential` and check if that's working?

Reply:
added